### PR TITLE
remove function overhead

### PR DESCRIPTION
**Pull request by lance-pyles**
add type hints

### **Describe your change:**

* [ ] Add an algorithm?
* [x] Fix a bug or typo in an existing algorithm?
* [ ] Documentation change?

### **Checklist:**
* [x] I have read [CONTRIBUTING.md](https://github.com/TheAlgorithms/Python/blob/master/CONTRIBUTING.md).
* [x] This pull request is all my own work -- I have not plagiarized.
* [x] I know that pull requests will not be merged if they fail the automated tests.
* [x] This PR only changes one algorithm file.  To ease review, please open separate PRs for separate algorithms.
* [x] All new Python files are placed inside an existing directory.
* [x] All filenames are in all lowercase characters with no spaces or dashes.
* [x] All functions and variable names follow Python naming conventions.
* [x] All function parameters and return values are annotated with Python [type hints](https://docs.python.org/3/library/typing.html).
* [x] All functions have [doctests](https://docs.python.org/3/library/doctest.html) that pass the automated testing.
* [x] All new algorithms have a URL in its comments that points to Wikipedia or other similar explanation.
* [x] If this pull request resolves one or more open issues then the commit message contains `Fixes: #{$ISSUE_NO}`.

### CODE REVIEW

File: maths/area.py [hunk 1]
```diff
@@ -14,7 +14,7 @@ def surface_area_cube(side_length: Union[int, float]) -> float:
     >>> surface_area_cube(3)
     54
     """
-    return 6 * pow(side_length, 2)
+    return 6 * side_length ** 2
```
Comment (cclauss) [inline maths/area.py:17]: Are we going to bounce back and forth every 48 hours?
https://github.com/TheAlgorithms/Python/pull/2223#pullrequestreview-453737047

File: maths/area.py [hunk 2]
```diff
@@ -5,7 +5,7 @@
 from typing import Union
```
Comment (cclauss) [inline maths/area.py]: Travis not happy... ./maths/area.py:5:1: F401 'typing.Union' imported but unused

Comment (lance-pyles) [reply maths/area.py]: I swear that line keeps coming back by itself :)

**Comment by cclauss** [comment]
Should I merge it quick before that import reappears?!?

**Comment by poyea** [comment]
How about we just leave it there? Or we can ignore this file

**Pull request merged**